packages/ckeditor5-image/lang/contexts.json

```diff
@@ -42,5 +42,8 @@
 	"Uploading image": "Aria status message indicating that the image is being uploaded. Example: 'Uploading image'.",
 	"Image upload complete": "Aria status message indicating that the image has been uploaded successfully. Example: 'Image upload complete'.",
 	"Error during image upload": "Aria status message indicating that an error has occurred during image upload. Example: 'Error during image upload'.",
-	"Image": "Label for the widget inserted by the image feature (as in 'insert image')"
+	"Image": "Label for the widget inserted by the image feature (as in 'insert image')",
+	"Back": "The label of the button that returns to the previous view.",
```

Comment:
This one is already here :point\_right: [https://github.com/ckeditor/ckeditor5/blob/ck/epic/17230-linking-experience/packages/ckeditor5-link/lang/contexts.json#L19](https://github.com/ckeditor/ckeditor5/blob/ck/epic/17230-linking-experience/packages/ckeditor5-link/lang/contexts.json#L19)

Maybe if it will be repeatable we should move bot to the core? [https://github.com/ckeditor/ckeditor5/blob/ck/epic/17230-linking-experience/packages/ckeditor5-core/lang/contexts.json](https://github.com/ckeditor/ckeditor5/blob/ck/epic/17230-linking-experience/packages/ckeditor5-core/lang/contexts.json)

WDYT?

Reply:
yep, good idea

Reply:
Done https://github.com/ckeditor/ckeditor5/pull/17546/commits/f61880e936558052592bb22c3442a4cd75f1e5cb